docs/swagger.yaml

```diff
@@ -1043,6 +1043,21 @@ paths:
       tags:
         - public
         - miner
+  /public/miners/storage/query/miner}:
```

Comment:
FYI: This will be automatically removed by the swagger automation script. You should put any of these changes to the handle function annotation.

Reply:
This was added by the swagger workflow on commit 811a8eb2838da3a3bb416d79535438a32c120e8d